src/get-glamor-classname.js

```diff
@@ -27,10 +26,22 @@ function extractGlamorStyles(className = '') {
 export default getGlamorClassName
 
 function getGlamorClassName(styles, props, cssOverrides, theme) {
-  const mappedArgs = styles.slice()
-  for (let i = mappedArgs.length; i--;) {
-    if (typeof mappedArgs[i] === 'function') {
-      mappedArgs[i] = mappedArgs[i](props, theme)
+  let className, current
+  const mappedArgs = []
+  const nonGlamorClassNames = []
+  for (let i = 0; i < styles.length; i++) {
```

Comment:
Is there a reason we're using a `for` loop instead of `forEach` or is it just preference?